crates/anemo/src/network/peer.rs

```diff
@@ -34,6 +35,10 @@ impl Peer {
         self.connection.peer_id()
     }
 
+    pub fn connection_stats(&self) -> ConnectionStats {
+        self.connection.stats()
+    }
+
```

Comment:
Hmm i wonder if there's a way to not need to make this a public dependency...

This is fine for now but we'll need to come back to this probably.